drltrace/drltrace_frontend.cpp

```diff
@@ -97,6 +97,12 @@ static droption_t<bool> op_print_ret_addr
 (DROPTION_SCOPE_CLIENT, "print_ret_addr", false, "Print library call's return address",
  "Print return addresses of library calls.");
 
+static droption_t<uint> op_all_args
+(DROPTION_SCOPE_CLIENT, "all_args", 2, "Number of library call arguments to print",
+ "Number of library call arguments to print. Specify 0 to disable. Drltrace will print "
+ "symbolic representation of arguments in case of known library call including all "
+ "known arguments.");
```

Comment:
I would suggest splitting into two options: one that controls the number of unknown args printed ("num_unknown _args" or sthg), and another that is either a max on args of any type ("max_args" or sthg) or maybe just a boolean controlling whether any arg printing is done (the max seems more useful).